tests/core/repository/test_fs_repository_v2.py

```diff
@@ -0,0 +1,151 @@
+
+        fetched_model = r._load(m.id)
+        assert m == fetched_model
+
+    @pytest.mark.parametrize(
+        "mock_repo,params",
+        [
+            (MockFSRepositoryV2, {"model": MockModel, "dir_name": "mock_model"}),
+        ],
+    )
+    def test_get_all(self, mock_repo, params):
+        objs = []
+        r = mock_repo(**params)
+        r._delete_all()
+
+        for i in range(5):
+            m = MockObj(f"uuid-{i}", f"Foo{i}")
+            objs.append(m)
+            r._save(m)
+        _objs = r._load_all()
+
+        assert len(_objs) == 5
+
+        for obj in _objs:
+            assert isinstance(obj, MockObj)
+        assert sorted(objs, key=lambda o: o.id) == sorted(_objs, key=lambda o: o.id)
+
+    @pytest.mark.parametrize(
+        "mock_repo,params",
+        [
+            (MockFSRepositoryV2, {"model": MockModel, "dir_name": "mock_model"}),
+        ],
+    )
+    def test_delete_all(self, mock_repo, params):
+        r = mock_repo(**params)
+        r._delete_all()
+
+        for i in range(5):
+            m = MockObj(f"uuid-{i}", f"Foo{i}")
+            r._save(m)
+
+        _models = r._load_all()
+        assert len(_models) == 5
+
+        r._delete_all()
+        _models = r._load_all()
+        assert len(_models) == 0
+
+    @pytest.mark.parametrize(
+        "mock_repo,params",
+        [
+            (MockFSRepositoryV2, {"model": MockModel, "dir_name": "mock_model"}),
+        ],
+    )
+    def test_delete_many(self, mock_repo, params):
+        r = mock_repo(**params)
+        r._delete_all()
+
+        for i in range(5):
+            m = MockObj(f"uuid-{i}", f"Foo{i}")
+            r._save(m)
+
+        _models = r._load_all()
+        assert len(_models) == 5
+        r._delete_many(["uuid-0", "uuid-1"])
+        _models = r._load_all()
+        assert len(_models) == 3
+
+    @pytest.mark.parametrize(
+        "mock_repo,params",
+        [
+            (MockFSRepositoryV2, {"model": MockModel, "dir_name": "mock_model"}),
+        ],
+    )
+    def test_search(self, mock_repo, params):
+        r = mock_repo(**params)
+        r._delete_all()
+
+        m = MockObj("uuid", "foo")
+        r._save(m)
+
+        m1 = r._search("name", "bar")
+        m2 = r._search("name", "foo")
+
+        assert m1 is None
+        assert m == m2
+
+    @pytest.mark.parametrize(
+        "mock_repo,params",
+        [
+            (MockFSRepositoryV2, {"model": MockModel, "dir_name": "mock_model"}),
+        ],
+    )
+    @pytest.mark.parametrize("export_path", ["tmp"])
+    def test_export(self, mock_repo, params, tmp_sqlite, export_path):
+        r = mock_repo(**params)
+
+        m = MockObj("uuid", "foo")
+        r._save(m)
+
+        r._export("uuid", export_path)
+        assert pathlib.Path(os.path.join(export_path, "mock_model/uuid.json")).exists()
+        with open(os.path.join(export_path, "mock_model/uuid.json"), "r") as exported_file:
+            exported_data = json._load(exported_file)
```

Comment:
```suggestion
            exported_data = json.load(exported_file)
```

Reply:
I did a mass replace when we changed the name from `load` to `_load`, this is a mistake.